docs/ml-guide.md

```diff
@@ -185,7 +185,7 @@ val training = sc.parallelize(Seq(
 // Create a LogisticRegression instance.  This instance is an Estimator.
 val lr = new LogisticRegression()
 // Print out the parameters, documentation, and any default values.
-println("LogisticRegression parameters:\n" + lr.explainParams() + "\n")
+println(s"LogisticRegression parameters:\n ${lr.explainParams()}")
```

Comment:
Remove space after `\n`. The last `\n` should be preserved to create an empty line after this block.